pywbem/cim_obj.py

```diff
@@ -3653,33 +3701,43 @@ class CIMQualifierDeclaration(_CIMComparisonMixin):
         This variable will never be `None`.
 
       tosubclass (:class:`py:bool`):
-        If not `None`, specifies whether the qualifier value propagates
-        to subclasses.
+        If not `None`, specifies the tosubclass flavor. This flavor defines
+        whether the qualifier value propagates to subclasses. In MOF this
+        flavor is controlled by the `tosubclass` and `restricted` flavor
+        keywords.
```

Comment:
We should say that for non-None values, this attribute specifies the flavor of an implicitly defined qualifier type, and that implicitly defined qualifier types are deprecated in DSP0004. This comment applies to all flavor attributes of this class.